### ISSUE TITLE
mandatory not working in IE

### ISSUE THREAD

**Issue by fherrick** [aamir-poswal/SurveyTrackerApp#66]
Muhammad set up a new survey which has some mandatory questions
http://public.watg.com/watgsurvey/survey/4

The “check” for the mandatory questions only works in Chrome. Muhammad is  able to submit the survey without any questions answered in IE.

**Comment by fherrick** [comment]
I just tried this survey in IE11 and when I click submit, I do get the message for mandatory question. 
I’m connected to the WATG network.
![1](https://cloud.githubusercontent.com/assets/11509212/10000591/0478d0f0-6052-11e5-858f-d768ddc6e8cc.jpg)

**Comment by fherrick** [comment]
Muhammad is not getting the messages 
His IE11 is a different version

![muhammads ie](https://cloud.githubusercontent.com/assets/11509212/10000812/42830c34-6053-11e5-950d-caa4e501c2fc.png)


IE11 felicia is using
![felicias ie](https://cloud.githubusercontent.com/assets/11509212/10000819/48273f98-6053-11e5-804d-b75f77332eb0.png)

**Comment by aamir-poswal** [comment]
@fherrick,

Issue is fixed, and code changes are pushed to git repo. @tkoseoglu, Please promote the changes to stage server.

below machine IE 11 has installed, and I unit tested against it. So everything working as expected. Also you can verify it by yourself on below dev server.

Machine IP Address 216.65.214.89
Username: strongvpn.admin
Pass: SVPN123!

**Comment by fherrick** [comment]
From: Muhammad Bajwa 
Sent: Thursday, September 24, 2015 10:32 AM
To: Tolga Koseoglu; Aamir
Cc: Felicia Herrick; Aamir Sajjad
Subject: RE: survey/poll

I went back and tried to submit the survey entry in Internet Explorer without answering any questions.

This time it is correctly prompting me to enter my responses and not letting me through without answering mandatory questions.

Thank you for fixing this issue.

Muhammad Bajwa | Senior Director of Information Systems
WATG | designing destinations
300 Spectrum Center Drive, Suite 500 | Irvine, CA 92618  USA
+1 949 574 8500 tel
watg.com



 
From: Tolga Koseoglu 
Sent: Thursday, September 24, 2015 10:19 AM
To: Aamir <aamir.pucit@gmail.com>; Muhammad Bajwa <mbajwa@watg.com>
Cc: Felicia Herrick <fherrick@watg.com>; Aamir Sajjad <asajjad@watg.com>
Subject: RE: survey/poll

The survey app has been updated on PROD.

Sent from Mail for Windows 10
